CMakeLists.txt

```diff
@@ -53,19 +55,18 @@ FetchContent_Declare(
     GIT_REPOSITORY https://github.com/p-ranav/argparse.git
 )
 
-FetchContent_Declare(
-    CRCpp
-    GIT_REPOSITORY https://github.com/d-bahr/CRCpp.git
-    GIT_TAG release-1.2.0.0
-)
-
 FetchContent_MakeAvailable(
     yaml-cpp
     json
     noise
     argparse
-    CRCpp
 )
+execute_process(COMMAND wget -q https://cdn.cubic-mc.com/${MINECRAFT_VERSION}/blocks-${MINECRAFT_VERSION}.json -O ${CMAKE_CURRENT_BINARY_DIR}/blocks-${MINECRAFT_VERSION}.json)
+execute_process(COMMAND wget -q https://cdn.cubic-mc.com/${MINECRAFT_VERSION}/blocks-${MINECRAFT_VERSION}.json.sha256 -O ${CMAKE_CURRENT_BINARY_DIR}/blocks-${MINECRAFT_VERSION}.json.sha256)
+execute_process(COMMAND cd ${CMAKE_CURRENT_BINARY_DIR} && sha256sum --check blocks-${MINECRAFT_VERSION}.json.sha256)
```

Comment:
I truly hope there's another way of doing this, because this is making want to commit arson...